File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    let minLux = 99, minLuy = 99;
    let maxRdx = 0, maxRdy = 0;
    
    const width = wallpaper[0].length;
    const height = wallpaper.length;
    
    for (let i = 0; i < height; i++) {
        for (let j = 0; j < width; j++) {
            if (wallpaper[i][j] == '#') {
                minLux = Math.min(minLux, i);
                maxRdx = Math.max(maxRdx, i);
                minLuy = Math.min(minLuy, j);
                maxRdy = Math.max(maxRdy, j);
            }
        }
    }
    
    return [minLux, minLuy, maxRdx+1, maxRdy+1];
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    width = len(wallpaper[0])
    height = len(wallpaper)
    x, y = [], []
    
    for i in range(height):
        for j in range(width):
            if wallpaper[i][j] == '#':
                x.append(i)
                y.append(j)
    
    return min(x), min(y), max(x)+1, max(y)+1